config/locales/zh-TW.yml

```
@@ -1,36 +1,43 @@
 ---
 zh-TW:
-  credentials_required:
+  credentials_required: 需要認證
   edit: 編輯
   failure_when_forbidden: 請確認 URL 或再次提交
   feed_latest: RubyGems.org | 最新 Gems
   feed_subscribed: RubyGems.org | 訂閱 Gems
-  footer_about_html: RubyGems.org 是 Ruby 社群的 Gem 套件管理服務，讓你能立即地發佈及安裝你的 Gem 套件，並且利用
-    API 查詢及操作可用 Gem 的詳細資訊。<br />現在就成為貢獻者，貢獻一己之力來改善本站。
+  footer_about_html:
+    RubyGems.org 是 Ruby 社群的 Gem 套件管理服務，讓您能立即地<a href="%{publish_docs}">發佈</a>及<a href="%{install_docs}">安裝</a>您的 Gem 套件，並且利用
+    <a href="%{api_docs}">API</a> 查詢及操作<a href="%{gem_list}">可用 Gem</a> 的詳細資訊。<br />現在就<a href="%{contributing_docs}">成為貢獻者</a>，貢獻一己之力來改善本站。
   footer_sponsors_html:
+    RubyGems.org 透過與廣大的 Ruby 社群合作而誕生。
+    <a href="https://www.fastly.com/">Fastly</a> 提供頻寬和 CDN 支援，
+    <a href="https://www.rubycentral.org/">Ruby Central</a> 支付設備費用，並為進行中的開發和營運工作提供資金。
+    <a href="%{sponsors_page}">進一步了解我們的贊助商和它們是如何合作的</a>。
   footer_join_rt_html:
+    我們需要您的幫助來資助開發者為保證 RubyGems.org 能夠順利運行所耗費的時間。
```

MINOR: 覺得這一整句有點太長了
這樣看能不能更容易理解一點：

```suggestion
    我們需要您的資助來幫助開發者有更多開發時間，讓 RubyGems.org 能夠持續穩定地運行下去。
```